sequelizeDatabase/sequelFunctions.js

```diff
@@ -167,6 +167,37 @@ deleteCompany = (CompanyID) => {
         })
     })
 }
+
+retrieveCurrentEmployeesOfCompany = (CompanyID) => {
+    return new Promise((resolve,reject)=>{
```

Comment:
Aaron has informed me that by returning a new promise, the backend has more control over when the promise ends.